File: Insert a node in a BST[10-September-2023]/Solution.py
#   >> PYTHON CODE <<

class Solution:
    #Function to insert a node in a BST. 
    def insert(self,root, Key):
        if(root is None): return Node(Key)
        
        if(Key>root.data): root.right=self.insert(root.right,Key)
        elif Key<root.data: root.left=self.insert(root.left,Key)

        return root

File: Insert a node in a BST[10-September-2023]/Solution.java
//  >> JAVA CODE <<

class Solution {
    // Function to insert a node in a BST.
    Node insert(Node root, int Key) {
        if(root==null) return new Node(Key);
        
        if(Key>root.data) root.right=insert(root.right,Key);
        else if(Key<root.data) root.left=insert(root.left,Key);
        
        return root;
    }
}